test/run_pass/test_os_network_interfaces.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+var os = require('os');
+
+console.log(os.networkInterfaces());
```

Comment:
You should assert the fields, just like what the Node.js does at https://github.com/nodejs/node/blob/master/test/parallel/test-os.js#L131-L172.

Reply:
Yes, I will